lnrpc/rpc.proto

```diff
@@ -1381,6 +1389,19 @@ message GetInfoResponse {
     map<uint32, Feature> features = 19;
 }
 
+message GetRecoveryInfoRequest {
+}
+message GetRecoveryInfoResponse {
+    // Whether the wallet is in recovery mode
+    bool recovery_mode = 1;
+
+    // Whether the wallet recovery progress is finished
+    bool recovery_finished = 2;
+
+    // The recovery progress
+    string progress = 3;
```

Comment:
perhaps we should expose this as a `float64`, and allow applications to format how they please?

Reply:
yep, changed